examples/gum/script/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("cargo:rustc-link-arg=-rdynamic");
```

Comment:
can this not be specified in the Cargo.toml?

Reply:
I'm not sure. I've not seen that done before. Happy with it to be changed if that's preferable.

Reply:
I'm not sure either. If it can be done without a build.rs, better to do so.

Reply:
Sure. The only other way I've seen linker flags set is with config.toml. But that applies to all crates in the workspace. It is intended to allow platform rather than crate specific configuration I believe.